lib/main.dart

```diff
@@ -1,12 +1,112 @@
 import 'package:flutter/material.dart';
+import 'data.dart';
 
 void main() => runApp(MyApp());
 
 class MyApp extends StatelessWidget {
-  @override
   Widget build(BuildContext context) {
     return MaterialApp(
-      home: Center(child: Text('Empty Shell for Art Museum')),
+      theme: ThemeData(primaryColor: Colors.teal[50]),
+      home: HomePage(),
+    );
+  }
+}
+
+class HomePage extends StatefulWidget {
+  @override
+  _HomePageState createState() => _HomePageState();
+}
+
+class _HomePageState extends State<HomePage> {
+  List<ObjectInfo> picData;
+
+  void initState() {
+    super.initState();
+    requestImages().then((images) {
+      setState(() => picData = images);
+    });
+  }
+
+  Widget build(BuildContext context) {
+    return Scaffold(
+      backgroundColor: Colors.teal[100],
+      appBar: AppBar(
+        title: Text('The Art Museum'),
+        backgroundColor: Colors.teal[50],
+      ),
+      body: Column(
+        children: [
+          Expanded(child: worksOfArt()),
+          RaisedButton(
```

Comment:
Placing the button at the end of the list might be cool, since it is only relevant when you have reached the end of the currently available images? Idk. Let me know what you think. :)